great_ape_safe/great_ape_safe.py

```diff
@@ -253,23 +256,63 @@ def _generate_tenderly_simulation(self, receipt, gas):
         """
         docs: https://www.notion.so/Simulate-API-Documentation-6f7009fe6d1a48c999ffeb7941efc104
         """
-        header = {"X-Access-Key": os.getenv("TENDERLY_ACCESS_KEY")}
+        header = {
+            "Content-Type": "application/json",
+            "X-Access-Key": os.getenv("TENDERLY_ACCESS_KEY"),
+        }
         api_url = f'https://api.tenderly.co/api/v1/account/{os.getenv("TENDERLY_USER")}/project/{os.getenv("TENDERLY_PROJECT")}/simulate'
         tx_payload = {
             "network_id": str(chain.id),
-            "block_number": chain.height,
-            "transaction_index": 0,
-            "from": Contract(self.address).getOwners()[0],
+            "from": receipt.sender.address,
             "to": self.address,
             "input": receipt.input,
-            "gas": gas,
+            "gas": 1_500_000 if gas == 0 else gas,
             "save": True,
             "save_if_fails": True,
+            # storage ref: https://github.com/safe-global/safe-contracts/blob/main/contracts/libraries/SafeStorage.sol
             "state_objects": {self.address: {"storage": {"0x04": "0x01"}}},
         }
         r = requests.post(api_url, headers=header, json=tx_payload)
-        r.raise_for_status()
+        if not r.ok:
```

Comment:
this is automatically included in calling `r.raise_for_status()`

Reply:
you're [right](https://github.com/psf/requests/blob/main/requests/models.py#L756), feels redundant, remove in 0b62fc678bf1d546ca6d6ca6ba7713fe28c32887